### ISSUE TITLE
  FAILED_PRECONDITION: Couldn't get ptxas version string: INTERNAL: Couldn't invok

### ISSUE THREAD

**Issue by JoreyYan** [sokrypton/ColabFold#384]
## Steps to Reproduce (for bugs)
I install and run this model in 3090 24GB, Driver 525m CUDA :12.0
I just could predict part of PDBs, and the memory of GPU is little used. 

## ColabFold Output (for bugs)
The issue just like :
actually, the memory-usage just is 264/24576 Mb
## Context
![image](https://user-images.githubusercontent.com/35975693/218804189-d9967057-800f-47fe-9ecc-fcae7bdafe97.png)

**Comment by toobiwankenobi** [comment]
the same issue for my setup (8 CPU, 1 GPU, 128GB RAM) 

I found a similar issue described [here](https://github.com/google/jax/discussions/6843). I installed `conda install -c nvidia cuda-nvcc`, which solved the issue for me. 

However, I then got a warning (not an error): 
```
W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:497] The NVIDIA driver's CUDA version is 11.6 which is older than the ptxas CU
DA version (12.0.140). Because the driver is older than the ptxas version, XLA is disabling parallel compilation, which may slow down compilation. You should update your NVIDIA driver or
 use the NVIDIA-provided CUDA forward compatibility packages.
```

**Comment by milot-mirdita** [comment]
Please you the localcolabfold installer. It will install cuda versions that are known to known to work. Cuda/jax can be very finicky about software versions.

**Comment by alirezaomidi** [comment]
I had the same issue with CUDA version 11.3. Updated to 11.4 and the problem went away. Also see https://github.com/sokrypton/ColabFold/issues/386#issuecomment-1434426774

**Comment by sokrypton** [comment]
yeah... unfortunately, `pip install jax[...` doesn't automatically identify the correct version of jax that is compatible with your cuda/cudnn version(s). You have to explicitly set it. For example:
```bash
pip install -q "jax[cuda11_cudnn805]" -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
